pkg/dryrun/testdata/test_config_status/output.txt

```diff
@@ -0,0 +1,20 @@
+# Status compare:
+.compliancyDetails[0].Compliant: Compliant does not match NonCompliant
+.compliancyDetails[0].conditions[0].message: hello not match does not match pods [nginx-pod-e2e] not found in namespace default
+.compliancyDetails[0].conditions[0] does not match any elements
+.compliancyDetails[0].conditions does not match the elements
+.compliancyDetails[0] does not match any elements
+.compliancyDetails does not match the elements
+.compliant: Compliant does not match NonCompliant
+.relatedObjects[0].compliant: NonCompliant does match NonCompliant
+.relatedObjects[0].object.apiVersion: v1 does match v1
+.relatedObjects[0].object.kind: Pod does match Pod
+.relatedObjects[0] is founded
+.relatedObjects is founded
+[1m Expected status does not match the actual status [0m
+
```

Comment:
Why does only the bold part have terminal escape codes? Shouldn't every line that is colored have some?